recipes/r-dse/meta.yaml

```
@@ -0,0 +1,87 @@
+{% set version = '2020.2-1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-dse
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/dse_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/dse/dse_{{ version }}.tar.gz
+  sha256: 6c7ac300a1cb70d458249feef1488f7f9e867e25b612a85281487f1d6dee199c
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('fortran') }}        # [not win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-setrng >=2004.4_1
+    - r-tfplot
+    - r-tframe >=2007.5_3
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-setrng >=2004.4_1
+    - r-tfplot
+    - r-tframe >=2007.5_3
+    - libopenblas
```

Please instead of libopenblas in run add libblas to the host section:
```suggestion
    - r-tframe >=2007.5_3
    - libblas
  run:
    - r-base
    - {{ native }}gcc-libs         # [win]
    - r-setrng >=2004.4_1
    - r-tfplot
    - r-tframe >=2007.5_3
```

See also https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mesa#blas